R/00classes.R

```diff
@@ -189,7 +189,11 @@ setMethod("$", "Message", function(x, name) {
 		"set" = function(...) set( x, ... ),
 		"fetch" = function(...) fetch(x, ... ),
 		"toString" = function(...) toString( x, ... ),
-		"toJSON" = function(...) toJSON( x, ... ),
+		"toJSON" = function(preserve_proto_field_names = FALSE, always_print_primitive_fields = FALSE, ...)
+		  toJSON(x,
+				 preserve_proto_field_names = preserve_proto_field_names,
+				 always_print_primitive_fields = always_print_primitive_fields,
+				 ... ),
```

Comment:
Indentation in this old file is all over place but never two chars.  I made it a little more resemblant of the existing form.

Reply:
It's not 2 spaces, that entire paragraph is indented with tabs, so I kept it this way for consistency. Different people and github will see different things. Everything else in the repo seem to be indented with spaces.

Reply:
I use the same editor I always used (and which you use too, as I recall) and it came up wrong as it at GitHub making a 2:0 score which is why I fixed it. 

It is an old project and different tools where used by different people at different times. I do not believe in wholesale reformatting and in general tend to remove mode-lines now and rely mostly on `.editorconfig` -- which I just noticed had not been copied into this repo.  I may change that.

Reply:
Done.  Appears to be a little better now, at least with an editor that groks it.